Source/Common/json.env

```diff
@@ -1,15 +1,33 @@
 #CODE ERROR : 203
 
 _t4dConvertToJson(){
+# Generated From $Tools4Dev_PATH/Engine/G4d/templates.env
+# v1.8.0
 ###### _t4dConvertToJson
-#   Will convert stoud into json file with indent
-#   Usage : curl <URL> | _t4dConvertToJson 'file.json'
-#   Usage : echo '{"isLastPage": true,"limit": 25,"size": 9,"start": 0,"values": [{"yes": 25},{"no": 25}]}' | _t4dConvertToJson 'file.json'
-#   Demo  : echo '{"isLastPage": true,"limit": 25,"size": 9,"start": 0,"values": [{"yes": 25},{"no": 25}]}' | _t4dConvertToJson 'file.json' && tail -n+1 file.json
+#   - Name:
+#       _t4dConvertToJson
+###
+#   - Synopsis:
+#       Will convert stoud into json file with indent
+###
+#   - Usage:
+#       * curl <URL> | _t4dConvertToJson 'file.json'
+#       * echo '{"isLastPage": true,"limit": 25,"size": 9,"start": 0,"values": [{"yes": 25},{"no": 25}]}' | _t4dConvertToJson 'file.json'
+###
+#   - Example:
+#       * echo '{"isLastPage": true,"limit": 25,"size": 9,"start": 0,"values": [{"yes": 25},{"no": 25}]}' | _t4dConvertToJson 'file.json' && tail -n+1 file.json
+###
+#   - Note:
+#       <Required>
+#       [Optionnal]
+#       {XXXXXXX}       = Value To Be Changed, if {URL} then replace {URL} with a valid URL
+###
 ###### DOCUMENTATION BLOC #####
+########### CODE BLOC #########
 
-python -mjson.tool &> $1
+python3 -mjson.tool &> $1
```

Comment:
this will broke very old versions of linux, the best way to do this is to check `t4dCheckCommand python3` before doing it. 

be default to avoid breaking change we need to always keep the old workaround instead of replacing it